package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stytch",
-  "version": "10.20.0",
+  "version": "10.21.0",
```

Comment:
I think we need a major version bump because of all the renaming that happened with the user types: `Name` -> `UserName`, `Email` -> `UserEmail` etc